app/src/main/java/com/ballboycorp/com/topik/SonarBugSample.kt

```diff
@@ -0,0 +1,19 @@
+package com.ballboycorp.com.topik
+
+interface Sample {
+    fun method()
+
+    fun anotherMethod()
+}
+
+class SampleImpl : Sample {
+    override fun method() {
```

Comment:
Add a nested comment explaining why this function is empty or complete the implementation.